src/ConfigR/BasicConfig.Dictionary.cs

```diff
@@ -57,7 +59,7 @@ public bool ContainsKey(string key)
 
         public bool Remove(string key)
         {
-            log.TraceFormat(CultureInfo.InvariantCulture, "Removing '{0}' from {1}", key, this.GetSource());
+            log.TraceFormat(string.Format(CultureInfo.InvariantCulture, "Removing '{0}' from {1}", key, this.GetSource()));

```

Comment:
See this: https://cmatskas.com/application-logging-with-liblog/ regarding lamdas :)

-----Original Message-----
From: "Adam Ralph" <notifications@github.com>
Sent: ‎08/‎10/‎2015 18:27
To: "config-r/config-r" <config-r@noreply.github.com>
Cc: "Christos Matskas" <christos.matskas@softwarelounge.co.uk>
Subject: Re: [config-r] Logging uprade to LibLog (#205)

In src/ConfigR/BasicConfig.Dictionary.cs:
> @@ -57,7 +59,7 @@ public bool ContainsKey(string key)
>  
>          public bool Remove(string key)
>          {
> -            log.TraceFormat(CultureInfo.InvariantCulture, "Removing '{0}' from {1}", key, this.GetSource());
> +            log.TraceFormat(string.Format(CultureInfo.InvariantCulture, "Removing '{0}' from {1}", key, this.GetSource()));
You don't need to do lazy loading if you remove the call to string.Format(), since then the arguments will be passed straight through to LibLog and it will only use them if the logger is enabled.
—
Reply to this email directly or view it on GitHub.